lib/assert.js

```diff
@@ -1052,6 +1052,9 @@ assert.doesNotMatch = function doesNotMatch(string, regexp, message) {
 
 assert.CallTracker = CallTracker;
 
+const Snapshot = require('internal/assert/snapshot');
+assert.Snapshot = Snapshot;
```

Comment:
This needs to be documented if it's being exposed publicly.

Reply:
I agreee. as I mentiond:

> there is still some work to be done, and I want to receive some initial feedback before proceeding